eng/scripts/InstallationCheck.ps1

```diff
@@ -0,0 +1,32 @@
+param (
+    [Parameter()]
+    [string] $ArtifactsDirectory,
+
+    [Parameter()]
+    [string] $Artifact,
+
+    [Parameter()]
+    [string] $PipelineWorkspace
+)
+
+Write-Host "dotnet new console"
+dotnet new console
+$localFeed = "$PipelineWorkspace/$ArtifactsDirectory-signed/$Artifact"
+Write-Host "dotnet nuget add source $localFeed"
+dotnet nuget add source $localFeed
```

Comment:
Does the source need to e added given that you are explicitly pass it to the restore command?